pyfolio/utils.py

```diff
@@ -108,29 +96,37 @@ def default_returns_func(symbol):
         Daily returns for the symbol.
          - See full explanation in tears.create_full_tear_sheet (returns).
     """
+    start = '1/1/1970' if start is None else start
+    end = datetime.now() if end is None else end
```

Comment:
The pandas already defaults to today in `web.get_data_yahoo` so this isn't necessary except for readability purposes.